src/Control/Lens/Plated.hs

```diff
@@ -96,6 +104,7 @@ module Control.Lens.Plated
 
   -- * Generics
   , gplate
+  , gplate1
   , GPlated
```

Comment:
Since we export `GPlated`, we should probably export `GPlated1` as well, right?

Reply:
You're right; fixed!